### PR TITLE
Add immosuchmaschine as provider

### PR DESCRIPTION
**Pull request by creichel**
What the title says :) 

Actually, this might resolve partly #23 since immosuchmaschine is also scraping ImmobilienScout24 and other pages

### CODE REVIEW

**Comment by adriankumpf** [comment]
Sweet, another awesome PR 🚀 I did not know this page.

Pagination seems to be broken for me though. The CSS selector for the button is different, but even changing it does not work 🤔 That's not necessarily a blocker because having that provider is already a great addition.  Just wanted to check if it works for you?

**Comment by creichel** [comment]
Ah that could be (sorry for the bad testing therefore). Let me check this again and update the PR accordingly. Thanks for the heads up!

File: lib/sources/immosuchmaschine.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    price: '.data_price span | removeNewline | trim',
+    size:
+      '.data_size dd | removeNewline | trim',
+    title: '.data_title h3 a | removeNewline | trim',
+    link: '.data_title h3 a@href | removeNewline | trim',
+    description: '.data_desc span | removeNewline | trim',
+    address: '.data_zipcity span | removeNewline | trim',
+    rooms: '.data_rooms dd | removeNewline | trim',
+  },
+  paginate: '#srchrslt-pagination .pagination-next@href',
```
Comment (adriankumpf) [inline lib/sources/immosuchmaschine.js:37]: With this the pagination works for me. Can you try it?

```suggestion
  paginate: '.link.next@data-href',
```

Comment (creichel) [reply lib/sources/immosuchmaschine.js:37]: Sorry for my late answer. I will check that out and will add it to this PR if it's working fine :)

**Comment by creichel** [comment]
Your suggestions were added 🙌 thanks for testing!

**Review by adriankumpf — APPROVED** [review]
Nice work! Thank you @creichel 🙌

**Pull request merged**